staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt.go

```diff
@@ -149,7 +386,7 @@ type requestManagement struct {
 }
 
 // TypedConfigObjectReference is a reference to a relevant config API object.
-// No namespace is needed because none of these objects is namespaced.
+// No namespace is needed because none of these objects is namespaces.
```

Comment:
"namespaced" was correct.

Reply:
Sorry, I'm not sure how these comments were changed I agree with "namespaced".  I will change them back.